crates/services/shared-sequencer/src/http_api.rs

```diff
@@ -87,8 +88,10 @@ pub async fn estimate_transaction(
     let request = SimulateRequest {
         tx_bytes: tx_bytes.to_string(),
     };
+    let path = "/cosmos/tx/v1beta1/simulate";
```

Comment:
We have this problem in all requests=D 
<img width="1272" alt="image" src="https://github.com/user-attachments/assets/2f83c0eb-f9d2-4f81-9e84-fccd1bea044d" />

Reply:
Ah.. I just found the first one. Didn't look farther.